tests/renderer/components/settings-execution-spec.tsx

```diff
@@ -71,4 +74,29 @@ describe('ExecutionSettings component', () => {
       ]);
     });
   });
+
+  describe('handleEnvironmentVariableChange()', () => {
+    it('handles new environment variables', async () => {
+      const wrapper = shallow(<ExecutionSettings appState={store} />);
+      const instance = wrapper.instance() as any;
+      await instance.handleEnvironmentVariableChange({
+        currentTarget: { value: 'ELECTRON_DEBUG_DRAG_REGIONS=1' },
```

Comment:
_nit_ not a blocker, but a suggestion for cases like this: instead of using the same magic string in two places, declare `ELECTRON_DEBUG_DRAG_REGIONS=1` as a local const and use it in the invocation and in the expect.

And if you name the variable `value` then the invocation becomes `currentTarget: { value }` which reads kind of pleasingly.